atlas/commands/bootstrap/templates/docker/Dockerfile.gateway.gotmpl

```diff
@@ -1,11 +1,12 @@
 # build the gateway binary
-FROM golang:1.10.0 as builder
+FROM golang:1.10.0 AS builder
+LABEL stage=gateway-intermediate
 WORKDIR /go/src/{{ .Root }}/{{ .Name }}
 COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o bin/gateway ./cmd/gateway
 
-# build the gateway binary
-FROM alpine:latest as runner
+# copy the gateway binary from builder stage; run the gateway binary
+FROM alpine:latest AS runner
 WORKDIR /bin
```

Comment:
Should we do an `apk update --no-cache && apk upgrade  --no-cache ` for our final image to make sure packages are up to date for security purposes, or is that not a thing?

Reply:
I don't see anything in the Docker best practices about updating packages for security purposes, so I don't think that's a thing.